src/audio/copier/copier.c

```diff
@@ -2044,6 +2056,7 @@ static const struct comp_driver comp_copier = {
 		.dai_ts_start			= copier_dai_ts_start_op,
 		.dai_ts_stop			= copier_dai_ts_stop_op,
 		.dai_ts_get			= copier_dai_ts_get_op,
+		.dai_get_hw_params		= copier_get_hw_params,
```

Comment:
same question, how to handle this in later copier module adapter interface? just keep it?